### PR TITLE
Feature/gfi tooltip

### PR DESCRIPTION
**Pull request by warm-coolguy**
## Summary

See commit message https://github.com/Dataport/polar/pull/38/commits/723c2ac3698e2d6abf7dc5481b79a769f5c4bae6.

## Instructions for local reproduction and review

Run the Meldemichel client and check whether GeoLocation and GFI are appropriately displayed.

### CODE REVIEW

File: packages/plugins/Gfi/src/store/actions.ts [hunk 1]
```diff
@@ -54,6 +57,48 @@ const actions: PolarActionTree<GfiState, GfiGetters> = {
...
+            .filter((feature) => feature instanceof Feature) as Feature[]
+
+          const coordinate = features.length
+            ? map.getCoordinateFromPixel(pixel)
+            : undefined
+
+          overlay.setPosition(coordinate)
+
+          // remove FeatureLikes
+          if (features[0]) {
```
[the remark this answers is not in the document]
Comment (warm-coolguy) [reply packages/plugins/Gfi/src/store/actions.ts]: https://github.com/Dataport/polar/pull/38/commits/3f81046c8f6e858202fde88c0a8b3ce16c58f784

Whoops.

File: packages/plugins/Gfi/src/store/actions.ts [hunk 2]
```diff
@@ -54,6 +57,48 @@ const actions: PolarActionTree<GfiState, GfiGetters> = {
...
+          }
+          const features = map
+            .getFeaturesAtPixel(pixel, {
+              layerFilter: (layer) => layer.get('id') === layerId,
+            })
+            .filter((feature) => feature instanceof Feature) as Feature[]
+
+          const coordinate = features.length
+            ? map.getCoordinateFromPixel(pixel)
+            : undefined
```
[the remark this answers is not in the document]
Comment (warm-coolguy) [reply packages/plugins/Gfi/src/store/actions.ts]: Ah, right. https://github.com/Dataport/polar/pull/38/commits/34c19ea9581ec00e67f7df41f6db16601d558591

**Comment by warm-coolguy** [comment]
> 🏓 @warm-coolguy

🏓 @dopenguin

**Comment by dopenguin** [comment]
🏓 @warm-coolguy 

I added `OverlayPositioning.BOTTOM_CENTER` as there was a type error but now the tests fail as they can't resolve `OverlayPositioning`.  
Do you want to fix the test or do we ignore the type error?

**Comment by warm-coolguy** [comment]
> 🏓 @warm-coolguy
> 
> I added `OverlayPositioning.BOTTOM_CENTER` as there was a type error but now the tests fail as they can't resolve `OverlayPositioning`. Do you want to fix the test or do we ignore the type error?

There is no type error, or the tests wouldn't have run in the first place. E.g. add `const break: string = 5` at the start of the file to see this effect. Seems to be an IDE issue, my setup doesn't report on a type error either. Furthermore, `'ol/OverlayPositioning'` is not a thing that exists, physically. See the node_modules folder, it's not there.

Seems the last time this existed was on ol@5: https://openlayers.org/en/v5.3.0/apidoc/module-ol_OverlayPositioning.html

Not sure why the IDE would know and use that type. 🤔 

🏓 @dopenguin

**Review by dopenguin — APPROVED** [review]
🚀 @warm-coolguy

**Pull request merged**